bluecellulab/stimulus/factory.py

```diff
@@ -561,3 +643,47 @@ def neg_cheops(
             + Empty(self.dt, duration=post_delay)
         )
         return result
+
+    def sinespec(
```

Comment:
I hope all the parameters and formulae in sinespec are the same as those that we have in [bluepyemodel ](https://github.com/openbraininstitute/BluePyEModel/blob/main/bluepyemodel/ecode/sinespec.py) ecode of the sinespec

Reply:
Yes, thanks to the power of copy/pasting!